web_responsive/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import inherited_res_users
```

Comment:
The file should me called just res_users as the [convention](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming) says.

Reply:
Done.

Reply:
Nice, thank you!